File: CloudMusic/src/main/java/com/ken/android/CloudMusic/Receiver/ToolBarReceiver.java
package com.ken.android.CloudMusic.Receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.ken.android.CloudMusic.Activity.MusicListActivity;
import com.ken.android.CloudMusic.Config;
import com.ken.android.CloudMusic.PlayerService;
import com.ken.android.CloudMusic.R;

/**
 * Created by axnshy on 16/7/31.
 */
public class ToolBarReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        //PlayerState代表播放状态
        if (intent.getAction() == Config.MUSIC_UPDATE) {
            // PlayerState = intent.getIntExtra("PlayerState", 0x01);
            /*switch (PlayerService.PlayerState) {
                *//*
                * 1:CIRCLE
                * 2:PREVIOUS
                * 3:PLAY
                * 4:NEXT
                *
                *
                *
                * *//*
                case 1: {
                    mediaPlay.setImageResource(R.drawable.play);
                    //Toast.makeText(context, "music is prepared", Toast.LENGTH_SHORT).show();
                    TopMusicDisplay.setText("");
                    break;
                }
                case 2: {
                    mediaPlay.setImageResource(R.drawable.pause);
                    this.getTopMusicDisplay.setText("正在播放: " + PlayerService.currentMusic.getMusicName());
                    //Toast.makeText(context, "music is playing", Toast.LENGTH_SHORT).show();
                    break;
                }
                case 3: {
                    mediaPlay.setImageResource(R.drawable.play);
                    TopMusicDisplay.setText(" ");
                    //Toast.makeText(context, "music is paused", Toast.LENGTH_SHORT).show();
                    break;

                }
            }*/
        }
    }

}

File: SmartPlay/src/main/java/com/luffy/smartplay/Receiver/ToolBarReceiver.kt
package com.luffy.smartplay.Receiver

import android.content.*
import com.luffy.smartplay.Config

/**
 * Created by axnshy on 16/7/31.
 */
class ToolBarReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        //PlayerState代表播放状态
        if (intent.action === Config.Companion.MUSIC_UPDATE) {
            // PlayerState = intent.getIntExtra("PlayerState", 0x01);
            /*switch (PlayerService.PlayerState) {
                */
            /*
                * 1:CIRCLE
                * 2:PREVIOUS
                * 3:PLAY
                * 4:NEXT
                *
                *
                *
                * */
            /*
                case 1: {
                    mediaPlay.setImageResource(R.drawable.play);
                    //Toast.makeText(context, "music is prepared", Toast.LENGTH_SHORT).show();
                    TopMusicDisplay.setText("");
                    break;
                }
                case 2: {
                    mediaPlay.setImageResource(R.drawable.pause);
                    this.getTopMusicDisplay.setText("正在播放: " + PlayerService.currentMusic.getMusicName());
                    //Toast.makeText(context, "music is playing", Toast.LENGTH_SHORT).show();
                    break;
                }
                case 3: {
                    mediaPlay.setImageResource(R.drawable.play);
                    TopMusicDisplay.setText(" ");
                    //Toast.makeText(context, "music is paused", Toast.LENGTH_SHORT).show();
                    break;

                }
            }*/
        }
    }
}